src/components/scene/wakelock.js

```diff
@@ -0,0 +1,15 @@
+var register = require('../../core/component').registerComponent;
+var Wakelock = require('../../../vendor/wakelock/wakelock');
```

Comment:
There's not much way to remove this component, just as position is a non-optional component.